conf/config-docs.yaml

```diff
@@ -0,0 +1,124 @@
+# NOTE: When using inc:${path}, the path will be interpreted as relative to /conf, with no possibility
```

Comment:
is `config-docs.yaml` documentation, or actual configuration?  If documentation, could it be a Markdown file, instead of YAML?